qiskit/synthesis/linear/linear_depth_lnn.py

```diff
@@ -267,7 +58,7 @@ def synth_cnot_depth_line_kms(mat: np.ndarray[bool]) -> QuantumCircuit:
     # Returns the quantum circuit constructed from the instructions
     # that we got in _optimize_cx_circ_depth_5n_line
     num_qubits = len(mat)
-    cx_inst = _optimize_cx_circ_depth_5n_line(mat)
+    cx_inst = optimize_cx_circ_depth_5n_line(mat)
     qc = QuantumCircuit(num_qubits)
```

Comment:
it would be nice to have all the code to construct the quantum circuit be entirely in rust.

Reply:
Done in 4902815e1